aws-janitor/resources/route53.go

```diff
@@ -96,7 +96,9 @@ func route53ResourceRecordSetsForZone(opts Options, logger logrus.FieldLogger, s
 			}
 			logger.Warningf("%s: deleting %T: %s", o.ARN(), rrs, *rrs.Name)
 			if !opts.DryRun {
-				toDelete = append(toDelete, o)
+				if _, ok := opts.SkipResourceRecordSetTypes[*rrs.Type]; !ok{
+					toDelete = append(toDelete, o)
+				}
 			}
 		}
 		return true
```

Comment:
(can't comment on line 182, 191)

"unable to delete" log lines seem to lack context about what record / zone caused the issue - would it make sense to add resource name / arn to the error message?